samples/bluetooth/central_hids/README.rst

```diff
@@ -40,10 +40,14 @@ Button 1:
    Toggle the CAPSLOCK LED on the connected keyboard using Write without response.
    This function is available only if the connected keyboard is set to work in Boot Protocol Mode.
 
+   When pairing/bonding, press this button to confirm the passkey value that is printed on the COM listener to pair/bond with the other device.
```

Comment:
I think we could skip using pairing/bonding, and just say pairing. The procedure is pairing, bonding includes pairing.

Reply:
fixed it